### DISCUSSION TITLE
What should be used as equivalent for ScrollablePane in v9

### DISCUSSION THREAD

**Discussion by martinguenther** [microsoft/fluentui#30598] [Q&A] [answered]
The `ScrollablePane`was an easy way to get scrollbars applied to elements, like large data tables in a container.

In v8
```
<ScrollablePane scrollbarVisibility={ScrollbarVisibility.auto}>
	<DetailsList
		items={items}
		...
	/>
</ScrollablePane>
```

With v9 `DataGrid` and v8 `ScrollablePane`

```
<ScrollablePane scrollbarVisibility={ScrollbarVisibility.auto}>
	<DataGrid
		items={items}
		...
	>
		<DataGridHeader>
		   ...
		</DataGridHeader>
		<DataGridBody<item>>
			...
		</DataGridBody>
	</DataGrid>
</ScrollablePane>
```

Is there an equivalent in v9 which I did not see, or is it recommended to use a `<div>` with appropriate CSS?

**Comment by smhigley** [comment, marked as answer]
Right now we do not have an equivalent component in v9. My recommendation would be what you mentioned -- a `<div>` with appropriate CSS (i.e. `overflow: auto` and potentially flex styles if you need a sticky header).

The "DetailsList Fixed Header without ScrollablePane" example in the v8 docs site is actually a good resource for sample scrolling styles without using scrollablepane: https://developer.microsoft.com/en-us/fluentui#/controls/web/scrollablepane

Let me know if that answers your question!
